lib/documents.js

```diff
@@ -98,9 +106,26 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
  * @returns {Promise<object>} An object with the registration record.
  */
 export async function register({
-  internalId, externalId, document, recipients, ttl, creator,
+  internalId, externalId, document, recipients, recipientChain, ttl, creator,
   tokenizer, externalIdHash, documentHash, creatorHash, newRegistration
 } = {}) {
+  assert.optionalArrayOfObject(recipients, 'recipients');
+  assert.optionalArrayOfArray(recipientChain, 'recipientChain');
+
```

Comment:
```suggestion
```

Reply:
Suggest removing extra whitespace lines here and below to keep the whole "validation block" together.